tools/run_tests/run_tests_matrix.py

```diff
@@ -377,62 +377,33 @@ def _create_portability_test_jobs(
             timeout_seconds=_CPP_RUNTESTS_TIMEOUT,
         )
 
-    # portability C on Windows 64-bit (x86 is the default)
+    # portability C & C++ on Windows 64-bit
     test_jobs += _generate_jobs(
-        languages=["c"],
+        languages=["c", "c++"],
         configs=["dbg"],
         platforms=["windows"],
-        arch="x64",
-        compiler="default",
+        arch="default",
+        compiler="cmake_ninja_vs2019",
         labels=["portability", "corelang"],
         extra_args=extra_args,
         inner_jobs=inner_jobs,
+        timeout_seconds=_CPP_RUNTESTS_TIMEOUT,
     )
 
-    # portability C on Windows with the "Visual Studio" cmake
+    # portability C and C++ on Windows with the "Visual Studio 2022" cmake
     # generator, i.e. not using Ninja (to verify that we can still build with msbuild)
     test_jobs += _generate_jobs(
-        languages=["c"],
+        languages=["c", "c++"],
         configs=["dbg"],
         platforms=["windows"],
-        arch="default",
-        compiler="cmake_vs2019",
+        arch="x64",
+        compiler="cmake_vs2022",
```

Comment:
do we not need vs2019 anymore?

Reply:
vs2019 is covered by `cmake_ninja_vs2019` above.